src/explorer/Exploration.cs

```diff
@@ -47,20 +44,20 @@ public Exploration(string dataSource, string table, IList<ColumnExploration> col
             }
         }
 
-        public static ExplorationStatus ConvertToExplorationStatus(TaskStatus status)
+        public override IEnumerable<ExploreMetric> PublishedMetrics
         {
-            return status switch
+            get
             {
-                TaskStatus.Canceled => ExplorationStatus.Canceled,
-                TaskStatus.Created => ExplorationStatus.New,
-                TaskStatus.Faulted => ExplorationStatus.Error,
-                TaskStatus.RanToCompletion => ExplorationStatus.Complete,
-                TaskStatus.Running => ExplorationStatus.Processing,
-                TaskStatus.WaitingForActivation => ExplorationStatus.Processing,
-                TaskStatus.WaitingToRun => ExplorationStatus.Processing,
-                TaskStatus.WaitingForChildrenToComplete => ExplorationStatus.Processing,
-                _ => throw new Exception("Unexpected TaskStatus: '{exploration.Status}'."),
-            };
+                return ColumnExplorations.Select(ce => new UntypedMetric(
+                    name: "column_metrics",
+                    metric: new
+                    {
+                        ce.Column,
+                        ColumnType = ce.ColumnInfo?.Type ?? DValueType.Unknown,
+                        ce.Status,
+                        Metrics = ce.PublishedMetrics.ToList(),
+                    }) as ExploreMetric);
```

Comment:
Remove unnecessary cast.
```suggestion
                    }));
```